docs/getting-started/binary.md

```diff
@@ -68,4 +66,4 @@ dragonfly --logtostderr --requirepass=youshallnotpass --cache_mode=true -dbnum 1
 
 #### Flag files
 
-Flags can be also provided from a configuration file with the `--flagfile <filename>` flag. The file should list one flag per line, with equal signs instead of spaces for key-value flags.
+To add flags from a configuration file, use the `--flagfile <filename>` flag. The file must list one flag per line. For key-value flags, an equals sign (`=`) replaces space characters (` `).
```

Comment:
I wouldn't say they "replace" space characters, as that's also the case for non-files flags (i.e. just command line).
Maybe something along the lines of "Use the format of `--flag_name=flag_value` (note the `=` character, using a space (` `) won't work)." ?